### ISSUE TITLE
Record Selector not reflecting values when PK is UUID

### ISSUE THREAD

**Issue by rajatvijay** [centerofci/mathesar#2715]
1. Set up sample data

```SQL
CREATE TABLE example_table (
  id UUID DEFAULT uuid_generate_v1mc() PRIMARY KEY,
  column1 TEXT,
  column2 INTEGER,
  column3 BOOLEAN
);
INSERT INTO example_table (column1, column2, column3) VALUES
('foo', 1, TRUE),
('bar', 2, FALSE),
('baz', 3, TRUE);
```

2. Run reflection
3. Go to `example_table` and open a record selector page for any record. 
4. Expect the `id` field to be filled
5. Observer the below screenshot
<img width="1512" alt="Screenshot 2023-03-20 at 6 07 07 AM" src="https://user-images.githubusercontent.com/11032856/226221675-e59235a5-277f-462e-b17b-224a64dfe5e9.png">

I also tested the BE API just to make sure that it works with UUID in the URL for fetching records data and it works fine: 
<img width="1501" alt="Screenshot 2023-03-20 at 6 18 07 AM" src="https://user-images.githubusercontent.com/11032856/226221987-928cee2a-59e1-40c5-952c-bb5cc31f666d.png">

The most probable culprit here: https://github.com/centerofci/mathesar/blob/76660f929c7d291c30003abde03a294b9be58e3f/mathesar_ui/src/routes/TableRoute.svelte#L38

**Comment by pavish** [comment]
I'm picking this up for the sake of time. @seancolsen could review it during Eastern Time.

**Comment by rajatvijay** [comment]
Closing this as its fixed by #2719
